packages/dashboard/src/components/resourceExplorer/components/panel.tsx

```diff
@@ -32,69 +41,99 @@ const PanelAssetPropertyDragHandle = ({ item }: { item: ExtendedPanelAssetSummar
 
   return collected.isDragging ? (
     <div className="resource-explorer-panel-asset-property-drag-handle-dragging" ref={dragPreview}>
-      <Icon name="expand" /> <span>{item.name}</span>
+      <PanelAssetProperty item={item} />
     </div>
   ) : (
     <div className="resource-explorer-panel-asset-property-drag-handle" ref={dragSource} {...collected}>
-      <Icon name="expand" />
+      <PanelAssetProperty item={item} />
     </div>
   );
 };
 
+interface DescribeAssetResponseWithChildAssets extends DescribeAssetResponse {
+  children: ExtendedPanelAssetSummary[];
+}
+
 export interface IotResourceExplorerPanelProps {
   panelItems: ExtendedPanelAssetSummary[];
   handlePanelItemClick: (item: ExtendedPanelAssetSummary) => void;
   messageOverrides: DashboardMessages;
+  asset: DescribeAssetResponseWithChildAssets | null;
 }
 
 export const IotResourceExplorerPanel: React.FC<IotResourceExplorerPanelProps> = ({
-  panelItems,
   handlePanelItemClick,
+  asset,
   messageOverrides,
 }) => {
   const handlePanelItemClickInner = (e: Event, item: ExtendedPanelAssetSummary) => {
     e.preventDefault();
     handlePanelItemClick(item);
   };
 
-  const PanelCell = ({ item }: { item: ExtendedPanelAssetSummary }) => {
-    if (item?.isHeader) {
-      return <Box variant="awsui-key-label">{item.name}</Box>;
-    }
+  const Header = ({ children }: { children: React.ReactNode }) => (
+    <Box variant="awsui-key-label" display="inline">
+      {children}
+    </Box>
+  );
 
-    if (item?.isAssetProperty) {
-      return <span>{item.name}</span>;
-    }
+  const Text = ({ children }: { children: React.ReactNode }) => <Box>{children}</Box>;
 
-    return (
+  const LinkItem = ({ item }: { item: ExtendedPanelAssetSummary }) => (
+    <div>
       <Link href="#" onFollow={(e) => handlePanelItemClickInner(e, item)}>
         {item.name}
       </Link>
-    );
-  };
+    </div>
+  );
 
-  const tableColumnDefinitions = [
-    {
-      id: 'drag',
-      header: null,
-      width: '45px',
-      cell: (item: ExtendedPanelAssetSummary) => item?.isAssetProperty && <PanelAssetPropertyDragHandle item={item} />,
-    },
-    {
-      id: 'variable',
-      header: '',
-      maxWidth: '100%',
-      cell: (item: ExtendedPanelAssetSummary) => <PanelCell item={item} />,
-    },
-  ];
+  const hasChildAssets = !!asset?.children?.length;
+  const hasProperties = !!asset?.assetProperties?.length;
+  const hasAlarms = !!asset?.assetCompositeModels?.length;
```

Comment:
the presence of `assetCompositeModels` doesn't necessarily mean these composite models are alarms. Here is a util that helps you get the alarm source property https://github.com/awslabs/iot-app-kit/blob/main/packages/source-iotsitewise/src/alarms/iotevents/util/getAlarmSourceProperty.ts

From there you can extract the id that you will need to describe the alarm model. You will need that to get the name of the alarm. Just like it is done here https://github.com/awslabs/iot-app-kit/blob/main/packages/source-iotsitewise/src/alarms/iotevents/siteWiseAlarmModule.ts#L53

so we should do a check like 
```
  assetModel.assetModelCompositeModels
    ?.filter(({ type }) => type === 'AWS/ALARM')
```

Reply:
Got it, thanks!